submissions/ArthurGorbenko/HTML5-Form/index.html

```diff
@@ -0,0 +1,487 @@
+                Your order
+              </h4>
+              <table class="form-header_info_table">
+                <thead>
+                  <th class="form-header_status_table_cell">ITEMS</th>
+                  <th class="form-header_status_table_cell">DETAILS</th>
+                  <th class="form-header_status_table_cell">PRICE</th>
+                  <th class="form-header_status_table_cell">DELIVERY</th>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td class="form-header_status_table_cell">2 documents</td>
+                    <td class="form-header_status_table_cell">2 languages</td>
+                    <td class="form-header_status_table_cell">$696.06</td>
+                    <td class="form-header_status_table_cell">48 hours*</td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </section>
+            <section class="wrapper_form-header_status_buttons">
+              <button
+                type="submit"
+                class="button submit_button form-header_submit_button"
+              >
+                Place order
+              </button>
+              <button type="button" class="button form-header_print_button">
+                Print a quote
+              </button>
+            </section>
+          <section class="form-header_info">
+            <div class="wrapper_form-header_info_content">
+              <h4 class="form-header_titles form-header_info_title">
+                Help center
+              </h4>
+              <p class="form-header_info_text">
+                Lorem ipsum dolor sit amet.Lorem ipsum dolor sit amet.Lorem
+                ipsum dolor sit amet. Lorem ipsum dolor sit amet.Lorem ipsum
+                dolor sit amet.Lorem ipsum dolor sit amet.
+              </p>
+              <div class="wrapper_form-header_info_contacts">
+                <h4>+971 (0) 555 555</h4>
+                <h5 class="form-header_info_email">
+                  <strong>help@blabla.com</strong>
+                </h5>
+              </div>
+            </div>
+          </section>
+        </section>
+        <article class="form-main">
+          <div class="form-header_radio">
+            <span class="form-header_radio_text"
+              >Select what you want to translate</span
+            >
+          </div>
+          <section class="form-radio_choice_type">
+            <input
+              type="radio"
+              name="select_document"
+              id="radio_website"
+              class="visually__hidden"
+            />
+            <label for="radio_website" class="form-radio_choice_type_label">
+              <span class="form-radio_choice_type_button">
+                <img
+                  src="images/website-icon.png"
+                  alt="website logo"
+                  class="form-radio_choice_type_image"
+                />
+                <span><b>Website</b></span>
```

Comment:
For visual style need to use CSS, avoid styling by HTML tags.

Reply:
Done.